### PR TITLE
Add scroll to top button

### PR DESCRIPTION
**Pull request by TheNikhilRaj**
@4darsh-Dev Added scroll to top button!
#80
Capture

### CODE REVIEW

**Comment by TheNikhilRaj** [comment]
Hlo @4darsh-Dev  it's done!

File: django-web-app/cogniguard/settings.py [hunk 1]
```diff
@@ -24,11 +24,13 @@
 # PJ_SECRET_KEY=os.getenv('PJ_SECRET_KEY','')
 
 # for vercel deployment
-SECRET_KEY = os.environ.get('PJ_SECRET_KEY')
+#SECRET_KEY = os.environ.get('PJ_SECRET_KEY')
+SECRET_KEY = "default-development-secret-key"
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/settings.py:28]: You have to undo the development server changes, read the readme carefully, 
this is causing deployment failure

change debug to false and set the environ secret key

File: django-web-app/cogniguard/settings.py [hunk 2]
```diff
@@ -90,14 +92,21 @@
...
+    #    'USER': os.environ.get("DB_USER"),
+     #   'PASSWORD': os.environ.get("DB_PASSWORD"),
+      #  'HOST': os.environ.get("DB_HOST"),
+       # 'PORT': os.environ.get("DB_PORT"),
+    #}
+#}
+
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/settings.py:95]: similarly here

File: django-web-app/cogniguard/urls.py [hunk 3]
```diff
@@ -19,22 +17,10 @@
     path('admin/', admin.site.urls),
     # path("logout/", RedirectView.as_view(url="/admin/logout/")),
     path("", include("home.urls")),
-
-    # for rendering reset password cnf template
-    path('password-reset-confirm/<uidb64>/<token>/',
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/urls.py:24]: Here password-reset-confirm and complete paths are missing, 
see the latest changes 
vercel deployment configuration for static URL patterns are also missing

File: django-web-app/cogniguard/urls.py [hunk 4]
```diff
@@ -19,22 +17,10 @@
...
-         name='password_reset_complete'),
-
-
-        
     
 ]
 # vercel deployment url configuration
 
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```
Comment (4darsh-Dev) [inline django-web-app/cogniguard/urls.py:40]: same issue

File: django-web-app/templates/index.html [hunk 5]
```diff
@@ -61,15 +88,48 @@ <h1 class="font-mid-large font-primary">
             </p>
             <p>Creating a sense of false scarcity or pressure to act quickly, like countdown timers or
                 limited-time offers.</p>
-            <a href="{% url 'home:know-about-dp' %}" class="btn-secondary">Know More</a>
+            <a href="{% url 'home:knowDp' %}" class="btn-secondary">Know More</a>
```
Comment (4darsh-Dev) [inline django-web-app/templates/index.html:91]: here the url is incorrect, check latest code

File: django-web-app/templates/index.html [hunk 6]
```diff
@@ -61,15 +88,48 @@ <h1 class="font-mid-large font-primary">
...
+        let pos = document.body.scrollTop || document.documentElement.scrollTop;
+        let calcHeight = document.documentElement.scrollHeight - document.documentElement.clientHeight;
+        let percentval = Math.round((pos * 100) / calcHeight);
+
+        if (pos > 100) {
+            scrollToTop.style.display = "flex";
+        } else {
+            scrollToTop.style.display = "none";
+        }
+
```
Comment (4darsh-Dev) [inline django-web-app/templates/index.html:121]: you have made changes in the index.html, 
button will only appear on home page, 
for it to appear on all pages, you should add it in base.html

**Review by 4darsh-Dev — CHANGES_REQUESTED** [review]
## Changes are required 
fix them. 
Vercel deployment is failing due to incorrect configurations

**Comment by TheNikhilRaj** [comment]
@4darsh-Dev Can you delete my changes because i want fresh start!

**Comment by 4darsh-Dev** [comment]
@TheNikhilRaj  you can sync your fork, 
either revert the commits or create a new PR 
I'll recommend to revert the commits and sync your fork then make fresh changes

**Pull request closed without merging**